src/main/java/com/salaboy/conferences/site/model/AgendaItem.java

```diff
@@ -1,17 +1,25 @@
 package com.salaboy.conferences.site.model;
 
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
 import java.util.Objects;
 import java.util.UUID;
 
+@Entity(name = "agenda_item")
 public class AgendaItem {
 
+    @Id
+    @GenericGenerator(name="system-uuid", strategy = "uuid")
+    @GeneratedValue(generator="system-uuid")
     private String id;
     private String title;
     private String author;
     private String day;
     private String time;
 
-
     public AgendaItem() {
         this.id = UUID.randomUUID().toString();
```

Comment:
@mcruzdev if you have the generated value, this UUID.random.. is not needed anymore..

Reply:
i was doubted about this, thank you ;)